openshift-knative-operator/pkg/monitoring/monitoring_serving_test.go

```diff
@@ -19,8 +19,15 @@ func TestLoadPlatformServingMonitoringManifests(t *testing.T) {
 		t.Errorf("Got %d, want %d", len(manifests), 1)
 	}
 	resources := manifests[0].Resources()
-	if len(resources) != 20 {
-		t.Errorf("Got %d, want %d", len(resources), 20)
+
+	// We create a service monitor and a service monitor service per deployment
+	// One clusterrolebinding for allowing tokenreviews, subjectaccessreviews
+	// to be used by kube proxy. All deployments share the same sa.
+	// Five RBAC resources from rbac-proxy.yaml.
+	expectedServingMonitoringResources := len(servingDeployments)*2 + 6
```

Comment:
not sure I follow why `+ 6`

Reply:
As above 5 RBAC resources from rbac-proxy.yaml. plus 1 clusterrolebinding for the common service account used for all deployments.

Reply:
same here for a comment on the math

Reply:
ok

Reply:
Same concern as in the comment above: The 6 actually already assumes that there a 5 services, so maybe `len(serviceDeployments) + 1` might be more appropriate ?

Reply:
Yes will update it ok.